### PR TITLE
Add types for a few more webpack plugins

### PR DESCRIPTION
**Pull request by dmnd**
- Links to documentation: https://webpack.js.org/plugins/
- Link to GitHub or NPM: https://github.com/webpack/webpack
- Type of contribution: addition

Other notes:

Added types for some webpack plugins I'm using. I only tried to describe
types as I'm using these plugins; hopefully the way I'm calling these
plugins is common enough to be useful for others.

### CODE REVIEW

**Comment by dmnd** [comment]
Oops, looks like I tested these wrong. Will fix.

File: definitions/npm/webpack_v4.x.x/flow_v0.104.x-/test_webpack_v4.x.x.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      moduleFilenameTemplate: 'file://[absolute-resource-path]',
+      fallbackModuleFilenameTemplate: 'file://[absolute-resource-path]?',
+      noSources: false,
+    }),
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.ContextReplacementPlugin(
+      /highlight\.js\/lib\/languages$/,
+      /^\.\/(javascript|typescript|xml)/
+    ),
+  ],
```
Comment (dmnd) [inline definitions/npm/webpack_v4.x.x/flow_v0.104.x-/test_webpack_v4.x.x.js]: Lines 46-68 are new. The rest are from the original test file.

**Comment by dmnd** [comment]
Ok, CI is fixed. Sorry about that: this is now ready for review.

File: cli/src/commands/install.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ export function setup(yargs: Yargs) {
       alias: 'o',
       describe: 'Overwrite an existing libdef',
       type: 'string',
-      demand: false
+      demand: false,
```
Comment (pascalduez) [inline cli/src/commands/install.js]: I would prefer if you could revert this change, just to keep the history focused.

Comment (dmnd) [reply cli/src/commands/install.js]: No worries; dropped that commit.
